src/Security/Authentication/AuthenticationSuccessHandler.php

```diff
@@ -110,23 +98,4 @@ private function triggerPostLoginHook(User $user): void
             $this->framework->createInstance($callback[0])->{$callback[1]}($user);
         }
     }
-
-    /**
-     * Stores the locale after a user has logged in.
-     *
-     * @param Request $request
-     * @param string  $locale
-     */
-    private function setLocale(Request $request, string $locale): void
-    {
-        if (null !== ($session = $request->getSession()) && $session->isStarted()) {
-            $session->set('_locale', $locale);
-        }
-
-        $request->setLocale($locale);
-        $this->translator->setLocale($locale);
```

Comment:
I understand that we must not initialize the translator with the user's language in the front end, because the page language is the only relevant language. But where are we initializing the translator with the page language now?

Reply:
https://github.com/contao/core-bundle/blob/master/src/Resources/contao/pages/PageRegular.php#L70

Reply:
Yeah, I meanwhile figured this out, too (see #1282).